### DISCUSSION TITLE
ayuda no  se que estoy haciendo mal

### DISCUSSION THREAD

**Discussion by Isaisin** [forkineye/ESPixelStick#742] [Q&A]
quiero flashear un esp 32 para usarña por arnet pero no me arroja el ip no se que hago mal
![Captura de pantalla 2024-02-27 030858](https://github.com/forkineye/ESPixelStick/assets/161449689/810354d0-c9e7-4f3d-9ecf-4a3f261b3f61)

**Comment by MartinMueller2003** [comment]
Not sure what version of flashtool you ended up with but there are a bunch of buttons missing, leading me to think you may have gotten a very old version. Please download this dist.zip file and try again.
https://github.com/MartinMueller2003/ESPixelStick/releases/download/dist/dist.zip

**Reply by Isaisin** [reply]
hola perdon ya lo descargue y me dice que le faltan archivos tendras algun tutorial o en algun lugar puedo ver el proceso correcto

**Comment by MartinMueller2003** [comment]
Sorry about that. You can ignore the missing files. Just acknowledge them. Some platforms got added and they were not fully defined. As long as your platform is present, you should have no problem using the tool

**Comment by MartinMueller2003** [comment]
1. download the dist.zip file
2. Unzip the file to a local directory
3. cd to the dist directory created in the unzip step (for me it is:  C:\z\dist\dist)
4. Double Click on the "ESPSFlashTool.jar" file

For me, I do not get the file errors (above issue is resolved in latest tools). 

If you are getting file errors then you have an unzip issue or if you are on a mac, you have file permissions issues. 
It should look like this after you load the tool:

![image](https://github.com/forkineye/ESPixelStick/assets/8608503/61fa14b8-d7e2-4ccc-9937-a0828370a472)
